backend/src/types/Game.ts

```diff
@@ -0,0 +1,77 @@
+import { GameCard } from './Cards';
+import { GameEvent } from './Events';
+
+export enum Role {
+  SHERIFF = 'SHERIFF',
+  VICE = 'VICE',
+  RENEGADE = 'RENEGADE',
+  OUTLAW = 'OUTLAW',
+  NONE = 'NONE'
+}
+
+export class Player {
+  constructor (id: String, name: String, maxHealth: Number) {

```

Comment:
Constructor should just be the name. maxHealth won't be known until a character and role are assigned, id should be automatically generated (we can use the shortid library)